unitree_nav_interfaces/srv/SetGait.srv

```diff
@@ -0,0 +1,2 @@
+uint8 gait    # idle = 0, trot = 1, trot_running = 2, climb_stairs = 3, trot_obstacle = 4
```

Comment:
Define these values as constants in the message type.

[Example](https://docs.ros.org/en/noetic/api/visualization_msgs/html/msg/Marker.html).